processing/src/main/java/io/druid/segment/virtual/ExpressionColumnValueSelector.java

```diff
@@ -75,4 +75,10 @@ public void inspectRuntimeShape(RuntimeShapeInspector inspector)
     inspector.visit("expression", expression);
     inspector.visit("bindings", bindings);
   }
+
```

Comment:
getDouble/Float/Long need getObject().isNull() and replaceWithDefault() checks

Reply:
expressions already have checks, no need to add redundant checks in expression selectors.

Reply:
Please add comments for readers, expecting asserts

Reply:
added.